Mathlib/LinearAlgebra/Eigenspace/Triangularizable.lean

```diff
@@ -13,29 +13,35 @@ import Mathlib.FieldTheory.IsAlgClosed.Spectrum
 
 This file contains basic results relevant to the triangularizability of linear endomorphisms.
 
-* Every linear operator on a vector space over an algebraically closed field has an eigenvalue.
-* The generalized eigenvectors span the entire vector space.
+## Main definitions / results
```

Comment:
I think `exists_eigenvalue` can also be added as item to this list. What do you think?

Reply:
Agreed; done.